modules/ztp-deleting-node-using-siteconfig.adoc

```diff
@@ -0,0 +1,94 @@
+// Module included in the following assemblies:
+//
+// * edge_computing/ztp-advanced-install-ztp.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ztp-deleting-node-siteconfig_{context}"]
+= Deleting a node by using the SiteConfig CR
+
+By using a SiteConfig custom resource (CR), you can delete and reprovision a node. 
+This method is more efficient than manually deleting the node.
+
+.Prerequisites
+
+* You have configured the hub cluster to generate the required installation and policy CRs.
+
+* You have created a Git repository in which you can manage your custom site configuration data. The repository must be accessible from the hub cluster and be defined as the source repository for the Argo CD application.
+
+
+.Procedure
+
+. Update the  `SiteConfig` CR to include the `bmac.agent-install.openshift.io/remove-agent-and-node-on-delete=true` annotation:
++
+[source,yaml]
+----
+apiVersion: ran.openshift.io/v1
+kind: SiteConfig
+metadata:
+  name: "cnfdf20"
+  namespace: "cnfdf20"
+spec:
+  Clusters:    
+    nodes:
+    - hostname: node6
+      role: "worker"
+      crAnnotations:
+        add:
+          BareMetalHost:
+            bmac.agent-install.openshift.io/remove-agent-and-node-on-delete: true
+# ...
+----
+
+. Suppress the generation of the `BareMetalHost` CR by updating the `SiteConfig` CR to include the `crSuppression.BareMetalHost` annotation, for example:
```

Comment:
```suggestion
. Suppress the generation of the `BareMetalHost` CR by updating the `SiteConfig` CR to include the `crSuppression.BareMetalHost` annotation:
```
For consistency within this doc, you might want to consider removing the "for example" here as well.